sql/countries.sql

```diff
@@ -0,0 +1,250 @@
+INSERT INTO countries (name, code) VALUES ('South Sudan', 'SSD');
+INSERT INTO countries (name, code) VALUES ('Spain', 'ESP');
+INSERT INTO countries (name, code) VALUES ('Sri Lanka', 'LKA');
+INSERT INTO countries (name, code) VALUES ('Sudan (the)', 'SDN');
+INSERT INTO countries (name, code) VALUES ('Suriname', 'SUR');
+INSERT INTO countries (name, code) VALUES ('Svalbard and Jan Mayen', 'SJM');
+INSERT INTO countries (name, code) VALUES ('Swaziland', 'SWZ');
+INSERT INTO countries (name, code) VALUES ('Sweden', 'SWE');
+INSERT INTO countries (name, code) VALUES ('Switzerland', 'CHE');
+INSERT INTO countries (name, code) VALUES ('Syrian Arab Republic', 'SYR');
+INSERT INTO countries (name, code) VALUES ('Taiwan (Province of China)', 'TWN');
+INSERT INTO countries (name, code) VALUES ('Tajikistan', 'TJK');
+INSERT INTO countries (name, code) VALUES ('Tanzania, United Republic of', 'TZA');
+INSERT INTO countries (name, code) VALUES ('Thailand', 'THA');
+INSERT INTO countries (name, code) VALUES ('Timor-Leste', 'TLS');
+INSERT INTO countries (name, code) VALUES ('Togo', 'TGO');
+INSERT INTO countries (name, code) VALUES ('Tokelau', 'TKL');
+INSERT INTO countries (name, code) VALUES ('Tonga', 'TON');
+INSERT INTO countries (name, code) VALUES ('Trinidad and Tobago', 'TTO');
+INSERT INTO countries (name, code) VALUES ('Tunisia', 'TUN');
+INSERT INTO countries (name, code) VALUES ('Turkey', 'TUR');
+INSERT INTO countries (name, code) VALUES ('Turkmenistan', 'TKM');
+INSERT INTO countries (name, code) VALUES ('Turks and Caicos Islands (the)', 'TCA');
+INSERT INTO countries (name, code) VALUES ('Tuvalu', 'TUV');
+INSERT INTO countries (name, code) VALUES ('Uganda', 'UGA');
+INSERT INTO countries (name, code) VALUES ('Ukraine', 'UKR');
+INSERT INTO countries (name, code) VALUES ('United Arab Emirates (the)', 'ARE');
+INSERT INTO countries (name, code) VALUES ('United Kingdom of Great Britain and Northern Ireland (the)', 'GBR');
+INSERT INTO countries (name, code) VALUES ('United States Minor Outlying Islands (the)', 'UMI');
+INSERT INTO countries (name, code) VALUES ('United States of America (the)', 'USA');
+INSERT INTO countries (name, code) VALUES ('Uruguay', 'URY');
+INSERT INTO countries (name, code) VALUES ('Uzbekistan', 'UZB');
+INSERT INTO countries (name, code) VALUES ('Vanuatu', 'VUT');
+INSERT INTO countries (name, code) VALUES ('Venezuela (Bolivarian Republic of)', 'VEN');
+INSERT INTO countries (name, code) VALUES ('Viet Nam', 'VNM');
+INSERT INTO countries (name, code) VALUES ('Virgin Islands (British)', 'VGB');
+INSERT INTO countries (name, code) VALUES ('Virgin Islands (U.S.)', 'VIR');
+INSERT INTO countries (name, code) VALUES ('Wallis and Futuna', 'WLF');
+INSERT INTO countries (name, code) VALUES ('Western Sahara*', 'ESH');
+INSERT INTO countries (name, code) VALUES ('Yemen', 'YEM');
+INSERT INTO countries (name, code) VALUES ('Zambia', 'ZMB');
+INSERT INTO countries (name, code) VALUES ('Zimbabwe', 'ZWE');
```

Comment:
Laziness, sorry. Cleaned up in d78a132e9b03392e26b504ec54714ec2fea1b56f. :blush: